<log>
  <summary verdicts='3' failures='2' errors='0'/>
  <section name='Test' method='verdict_assert_kind_of?' type='fail'>
    <section name='Args=[Array, {}]'>
      <verdict method='verdict_assert_kind_of?' outcome='failed' id='0' message='Message 0'>
        <exp_value>
          Array
        </exp_value>
        <act_value/>
        <exception class='Minitest::Assertion'>
          <message>
            Expected {} to be a kind of Array, not Hash.
          </message>
          <backtrace/>
        </exception>
      </verdict>
    </section>
    <section name='Args=[Hash, []]'>
      <verdict method='verdict_assert_kind_of?' outcome='failed' id='1' message='Message 1'>
        <exp_value>
          Hash
        </exp_value>
        <act_value>
          []
        </act_value>
        <exception class='Minitest::Assertion'>
          <message>
            Expected [] to be a kind of Hash, not Array.
          </message>
          <backtrace/>
        </exception>
      </verdict>
    </section>
  </section>
  <verdict method='verdict_assert_equal?' outcome='passed' id='__error_count__'>
    <exp_value>
      0
    </exp_value>
    <act_value>
      0
    </act_value>
  </verdict>
</log>
